packages/pg-pool/package.json

```
@@ -34,6 +34,6 @@
     "pg-cursor": "^1.3.0"
   },
   "peerDependencies": {
-    "pg": ">5.0"
+    "pg": ">8.0"
```

`semver.validRange` confirms that this means “at least 8.1.0”.

```suggestion
    "pg": ">=8.0"
```